shell/src/main/java/alluxio/shell/command/MountCommand.java

```diff
@@ -83,6 +86,21 @@ public Options getOptions() {
   @Override
   public int run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
+    if (args.length == 0) {
+      Map<String, MountPointInfo> mountTable = mFileSystem.getMountTable();
+      String leftAlignFormat = " %-20s  %-3s %-60s %-8s %-15d %-15d %-5b %-5b   ";
```

Comment:
Because I designed the last `%-5b` corresponds to `shared` and following is the `properties` map printed by `println`. I was not sure how to print map using formatter, so I decided to use `println` in the end.